### PR TITLE
Add action sections to scorecard page linking to resources and national map pages

### PR DESCRIPTION
**Pull request by justinebreuch**
## Description

Addresses: Add share / additional resources section to scorecard page and Add Actions to Take section

### New

Adds ActionSection component that we need for the scorecard and error page. Includes a header, subheader and CTA button. 

### Changed

Moved footer out of landing page into all pages. The map page needs to be made larger to fit the screen in a separate PR. 

## Testing and Reviewing

See screenshots
Screen Shot 2022-08-12 at 12 44 37 PM
Screen Shot 2022-08-12 at 12 44 46 PM

### CODE REVIEW

**Comment by kailamjeter** [comment]
Does this cover the explanation of the score section? looks like it in the screenshot

File: client/src/components/ActionSection.vue [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  gap: 20px;
+  height: var(--height);
+  padding: 20px;
+}
+
+.outline-button {
+  color: var(--button-color)
+}
+
+
```
Comment (kailamjeter) [inline client/src/components/ActionSection.vue:66]: nit: remove whitespace here

File: client/src/components/PageFooter.vue [hunk 2]
```diff
@@ -39,6 +39,7 @@ export default defineComponent({
 
 <style scoped>
 .landing-page-footer {
```
Comment (kailamjeter) [inline client/src/components/PageFooter.vue:41]: can you remove 'landing' from the classnames here too

**Review by kailamjeter — APPROVED** [review]

File: client/src/components/ActionSection.vue (hunk 1 above)
Comment (justinebreuch) [reply client/src/components/ActionSection.vue:66]: Done!

File: client/src/components/PageFooter.vue (hunk 2 above)
Comment (justinebreuch) [reply client/src/components/PageFooter.vue:41]: Nice catch

**Comment by justinebreuch** [comment]
> screenshot

I just refreshed the text based on mocks but that section was there before

**Pull request merged**